src/textual/widgets/_input.py

```diff
@@ -284,6 +284,29 @@ def control(self) -> Input:
             """Alias for self.input."""
             return self.input
 
+    @dataclass
+    class Blur(Message):
```

Comment:
Let's call it `Blurred`, which is more consistent with existing messages, and avoids confusion with the existing `Blur` message.